kubernetes/longrun-test/deploy.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+show_help () {
+cat << EOF
+
+usage: $0 [-a] [-g] [-d] [-t] [-k] [-m]
+
+   -a : Install the entire setup.
+
+   -g : Install the latest golang.
```

Comment:
script appears to install a static version of golang.

Reply:
I would ideally want to just pull the latest golang without mentioning a version.

I found this:
```
git clone https://github.com/udhos/update-golang 
cd update-golang 
sudo ./update-golang.sh
```

Maybe worth a try.